action.yml

```diff
@@ -27,7 +32,7 @@ runs:
     - uses: actions/configure-pages@v2
     - uses: actions/upload-pages-artifact@v1
       with:
-        # Upload entire repository
+        # By default, upload the docs directory
```

Comment:
Nice catch 👍

Reply:
cherry-picked as https://github.com/kachick/deploy-yard-to-pages/commit/c6bddace4c6bf991e320e8694c31dc4ff1a5c231